cmake_modules/commonSetup.cmake

```diff
@@ -113,6 +113,13 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
   #"cmake -DEXAMPLEPLUGIN=ON <path-to/HPCC-Platform/>" will configure the plugin makefiles to be built with "make".
   option(EXAMPLEPLUGIN "Create a package with ONLY the exampleplugin plugin" OFF)
 
+  option(MAKE_CONFIGURATOR "Build Configurator" ON)
+  option(CONFIGURATOR_LIB "Build Configurator static library (.a)" OFF)
```

Comment:
A lib is valid target type.  This would be for an application linking against the Configurator and interfacing with its API.